libs/utils/WordPressUtils/build.gradle

```diff
@@ -4,8 +4,8 @@ buildscript {
         google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.0.1'
-        classpath 'com.novoda:bintray-release:0.5.0'
+        classpath 'com.android.tools.build:gradle:3.1.0'
```

Comment:
Same as for the login subtree, let's update the gradle wrapper to `4.4`, and `.travis.yml` to use `android-27` and build tools `27.0.3` here so this can build on its own.